sbi/utils/user_input_checks_utils.py

```diff
@@ -9,6 +9,19 @@
 from torch.distributions import Distribution, constraints
 
 
+def get_distribution_parameters(dist, device):
+    '''Used to get the tensors of the parameters in torch distributions.
```

Comment:
Use """ for docstring.

And a empty line before the second line of the docstring (otherwise the docs do not handle it correctly)